help_to_heat/frontdoor/interface.py

```diff
@@ -50,7 +51,7 @@ class FindAddressesSchema(marshmallow.Schema):
 
 
 class GetAddressSchema(marshmallow.Schema):
-    uprn = marshmallow.fields.Integer()
+    uprn = marshmallow.fields.String()
```

Comment:
This can actually go back to being an integer, I'll add a new class called `GetEpcSchema` and use accordingly

Reply:
UPRNs should probably be strings anyway (same reasoning as something like phone numbers; it's made of digits, but we're not intending to do maths on it).